tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP TABLE IF EXISTS conditions CASCADE;
+NOTICE:  drop cascades to table _timescaledb_internal.compress_hyper_2_4_chunk
+CREATE TABLE conditions(
```

Comment:
The conditions table already existed. Can't we reuse it or at least just truncate it?

Reply:
If tests are changed then there can be odd differences depending on setup. Safer and more stable to just drop the table and all it's dependent objects and recreate it in a known state.